lightning/log/testlogger.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap/zaptest"
+)
+
+// MakeTestLogger creates a Logger instance which produces JSON logs.
+func MakeTestLogger() (Logger, *zaptest.Buffer) {
```

Comment:
Is the `MakeTestLogger` function only used in the test?
if so, maybe we don't export this function will be better.

Reply:
Yes it is only used for testing. But it is used in two different packages (`lightning/mydump/csv_parser_test.go` and `lightning/restore/restore_test.go`), so this function cannot be private.